services/iplant-groups/src/iplant_groups/clients/grouper.clj

```diff
@@ -188,24 +187,24 @@
 ;; Group membership listings.
 
 (defn- group-membership-listing-error-handler
-  [group-id error-code {:keys [body] :as response}]
+  [group-name error-code {:keys [body] :as response}]
   (log/warn "Grouper request failed:" response)
   (let [body    (service/parse-json body)
         get-grc (fn [m] (-> m :WsGetMembersResults :results first :resultMetadata :resultCode))]
     (if (and (= error-code ce/ERR_REQUEST_FAILED) (= (get-grc body) "GROUP_NOT_FOUND"))
```

Comment:
I don't know if this is worth doing until we have an actual shipped feature but at some point it may be worth going through and checking if the errors grouper throws are more sensible (or at least, different) than the ones it did when we were using UUIDs. When I was originally writing some of these handlers (not this one, I think, but others), sometimes it just threw hibernate errors or such, but it might do better with group names.